composer.json

```
@@ -20,7 +20,7 @@
         "silverstripe/framework": "^4.10",
         "silverstripe/admin": "^1",
         "silverstripe/versioned": "^1",
-        "symfony/yaml": "~3.2"
+        "symfony/yaml": "~3.2 || ^4"
```

Can we do? Not that it makes much difference, but would be closer to all the other caret constraints.
```suggestion
        "symfony/yaml": "^3 || ^4"
```